cmake/app/boilerplate.cmake

```diff
@@ -411,6 +401,18 @@ foreach(root ${BOARD_ROOT})
           ${shield_dir}/${s_dir}/boards/${s}/${BOARD}.conf
           )
       endif()
+
+      # if shield config flag is on, add shield overlay to the shield overlays
+      # list and dts_fixup file to the shield fixup file
+      list(APPEND
+        shield_dts_files
+        ${shield_dir}/${s_path}
+        )
```

Comment:
Why is this code moved all the way to the end ?
It only appends to `shield_dts_files` so it should be grouped together with the other lines that appends to `shield_dts_files`

Reply:
ok, fixed